ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/customized-resource/cucushift-installer-rehearse-vsphere-ipi-customized-resource-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: cucushift-installer-rehearse-vsphere-ipi-customized-resource
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-vsphere-ipi-customized-resource-provision
+    post:
+    - chain: cucushift-installer-rehearse-vsphere-ipi-deprovision
+    env:
+      COMPUTE_NODE_CPU: "12"
+      COMPUTE_NODE_DISK_SIZE: "150"
+      COMPUTE_NODE_MEMORY: "10240"
+      COMPUTE_NODE_REPLICAS: "2"
+      CONTROL_PLANE_CPU: "12"
+      CONTROL_PLANE_DISK_SIZE: "150"
+      CONTROL_PLANE_MEMORY: "24576"
```

Comment:
Does this mean it will take 24G memory for each node? If possible, can it be lower a bit.

Reply:
ok

Reply:
@WenXinWei I saw you changed back to 24G memory from 10G, so 10G does not work, right?

Reply:
yes. the default value is 16G. seems installation will failed if memory less than 16G.so i change back to 24G.

Reply:
@WenXinWei  Can  you also help update the default value in `ipi-conf-vsphere-customized-resource` with the same values.